x-pack/test/stack_functional_integration/apps/ccs/ccs_discover.js

```diff
@@ -203,5 +224,51 @@ export default ({ getService, getPageObjects }) => {
         expect(hitCount).to.be.lessThan(originalHitCount);
       });
     });
+
+    it('should add detection rule', async () => {
```

Comment:
We don't need to add a test to create a rule, the execution of the rule should be part of it since we need it to perform the assertion.

Reply:
Should this reasoning be applied also to the index pattern creation then?

Reply:
That would be great! Take a look to my comment above.